docs/sphinx/about/whats-new.txt

```diff
@@ -1,61 +1,79 @@
 Version history
 ===============
 
-5.2.0-m5 (2016 July 7)
------------------------
+5.2.0 (2016 August 18)
+----------------------
 
 Java format support improvements are listed below.
 
 †Denotes a major breaking change to the reader (typically modification of core
 metadata). Code changes or re-import may be necessary in ImageJ/FIJI and
 OMERO.
 
-* added support for :doc:`Becker & Hickl .spc FIFO </formats/becker-hickl-fifo>` data
+* added support (and public sample files) for
+  :doc:`Becker & Hickl .spc FIFO </formats/becker-hickl-fifo>` data
 * added support for :doc:`Princeton Instruments .spe </formats/princeton-instruments-spe>` data
 * bug fixes for many formats including:
-   - Tiff
-       - fixed integer overflow to read resolutions correctly
-       - fixed handling of tiled images with tile width less than 64
-   - PNG writing
-   - Zeiss ZVI†
-       - reworked image ordering calculation to allow for tiles
-   - Leica LIF
-       - fixed incorrect plane offsets for large multi-tile files
-   - MRC and Spider
-       - fixes for format type checking
-   - OME-XML
-       - fixed metadata store
    - CellSens VSI†
        - fixes for correctly reading dimensions
-   - ICS writing
-       - fixed dimension population for split files
    - FlowSight
        - fixes to infer channel count from channel names (thanks to Lee
          Kamentsky)
    - Hamamatsu VMS†
        - fixed dimensions of full-resolution images
-   - PicoQuant
-       - updated reader to always buffer data
-   - Zeiss LSM
-       - fixed Plane population errors
+   - ICS writing
+       - fixed dimension population for split files
+   - Kodak BIP
+       - fixed handling of CCD temperature stored in hexadecimal
+   - Leica LIF
+       - fixed incorrect plane offsets for large multi-tile files
+   - LiFlim
+       - fixed ExposureTime check and units usage
+   - MRC and Spider
+       - fixed format type checking
+   - Micro-Manager
+       - fixed handling of large datasets saved as image stacks and split
+         over multiple files
    - Nifti
        - fixed planeSize to prevent crashes when loading large files (thanks
          to Christian Niedworok)
-   - LiFlim
-       - fixed ExposureTime check and units usage
-   - SDT
-       - performance improvements for loading of large files
+       - added support for gzipped compressed .nii.gz files
```

Comment:
Although the PR became larger, I would add a thanks to Eric Barnhill who initiated the first contribution via https://github.com/openmicroscopy/bioformats/pull/2508/commits/e4c753d6d1f34c5486fd5229b15c19a6c1729787